benchmark/modules/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Klepsydra Technologies AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set(PROJ_NAME kpsr_${PROJ_MAIN_NAME})
```

Comment:
when you rename this folder to vision_ocv, rename this to `kpsr_vision_${PROJ_MAIN_NAME}`